### PR TITLE
Add a check to the "levels" argument supplied for //asc and //desc

### PR DESCRIPTION
**Pull request by dordsor21**
Feel free to commit separately with a different implementation, just keeping this the same as the quick FAWE patch. Can change downstream to whatever you guys implement as well.

### CODE REVIEW

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/NavigationCommands.java [hunk 1]
```diff
@@ -75,6 +77,8 @@ public void unstuck(Player player) throws WorldEditException {
     public void ascend(Player player,
                        @Arg(desc = "# of levels to ascend", def = "1")
                            int levels) throws WorldEditException {
+        World world = player.getWorld();
+        checkArgument(levels >= 1 && levels <= (world.getMaxY() - world.getMinY()), "1 <= levels <= world height");
```
Comment (octylFractal) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/NavigationCommands.java:81]:
```suggestion
        checkArgument(1 <= levels && levels <= (world.getMaxY() - world.getMinY()), "1 <= levels <= world height");
```
This makes it look more like the error message, and I prefer to have bounds done this way.

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/NavigationCommands.java [hunk 2]
```diff
@@ -98,6 +102,8 @@ public void ascend(Player player,
     public void descend(Player player,
                         @Arg(desc = "# of levels to descend", def = "1")
                             int levels) throws WorldEditException {
+        World world = player.getWorld();
+        checkArgument(levels >= 1 && levels <= (world.getMaxY() - world.getMinY()), "1 <= levels <= world height");
```
Comment (octylFractal) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/NavigationCommands.java:106]: Ditto here.
```suggestion
        checkArgument(1 <= levels && levels <= (world.getMaxY() - world.getMinY()), "1 <= levels <= world height");
```

**Review by octylFractal — CHANGES_REQUESTED** [review]
Minor styling fix, and would you mind extracting this to a common method so if it changes both places get updated?

**Comment by dordsor21** [comment]
For reference: IntellectualSites/FastAsyncWorldEdit#1718

**Pull request closed without merging**